game/src/js/scenes/DungeonScene.js

```diff
@@ -67,10 +68,28 @@ export default class DungeonScene extends BaseScene {
 
   create() {
     this.spawn(0,0);
-
+    this.delay = 50;
+    this.enemyList = [];
+    this.virtualTime = 0;
   }
 
   update() {
+    --this.delay;
+    ++this.virtualTime;
+    if (this.delay <= 0) {
+      var newMonster = new FireMonster(this, 250 * (Math.random() - 0.5), 250 * (Math.random() - 0.5));
```

Comment:
This should be instantiating the `Character` class, perhaps with either a random character type of having probabilities for spawning each character type.

Reply:
Yes I know that's ideal, I just didn't really see how to do that.